fe1-web/parts/lao/organizer/RollCallOpened.tsx

```diff
@@ -34,6 +37,12 @@ const RollCallOpened = () => {
   const navigation = useNavigation();
   const [attendees, updateAttendees] = useState(new Set<string>());
```

Comment:
it seems like you could now directly store PublicKey's here.

Reply:
True, I changed it.